google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClient.java

```diff
@@ -108,6 +119,17 @@ static AdminClient create(AdminClientSettings settings) throws ApiException {
    */
   ApiFuture<Subscription> createSubscription(Subscription subscription);
```

Comment:
`default ApiFuture<Subscription> createSubscription(Subscription subscription) { createSubscription(subscription, OffsetLocation.END); }`

Reply:
Also modify the comment to make it clear where this creates the subscription

Reply:
Done. This is ready for another look. Thanks!